src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -41,9 +45,15 @@ public Task getTask(Long taskId) {
 
     public Task addTask(Task task) {
         User user = authService.getLoggedUser();
+        ZonedDateTime now = ZonedDateTime.now(ZoneId.of("UTC")).truncatedTo(ChronoUnit.SECONDS);
```

Comment:
how did this code end up here?

Reply:
I added it manually because i need it to test statistic methods.

Reply:
Got it, I thought it ended up here by mistake